src/concrete/ml/sklearn/base.py

```diff
@@ -96,6 +101,9 @@
 # However, for internal testing purposes, we retain the capability to disable this feature
 os.environ["TREES_USE_ROUNDING"] = "1"
 
+# By default, the decision of the tree ensembles is made in clear
+os.environ["TREES_USE_FHE_SUM"] = "0"
```

Comment:
should't we consider what a user may have configured in their env ? e.g. 

`os.environ["TREES_USE_FHE_SUM"] = os.environ.get("TREES_USE_FHE_SUM", 0)`

Reply:
agree, and same should go `s.environ["TREES_USE_ROUNDING"] = "1"` I guess

Reply:
yes, do the same for that one too!